packages/@aws-cdk/aws-lambda/lib/lambda-insights.ts

```diff
@@ -88,47 +85,5 @@ function getVersionArn(context: IResolveContext, insightsVersion: string): strin
   }
 
   // Otherwise, need to add a mapping to be looked up at deployment time
-
-  /**
-   * See this for the context as to why the mappings are the way they are
-   * https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/mappings-section-structure.html
-   *
-   * Mappings have to have a structure like this, and no functions can be used inside them:
-   * <Alphanumeric only>
-   * - <Can be non-alphanumeric>
-   * -- { <alphanumeric>: "value1"},
-   * -- { <alphanumeric>: "value2"}
-   *
-   * So we cannot have an otherwise ideal mapping like this, because '1.0.98.0' is non-alphanumeric:
-   * LambdaInsightsVersions
-   * - us-east-1
-   * -- {'1.0.98.0': 'arn1'},
-   * -- {'1.0.89.0': 'arn2'}
-   *
-   * To get around this limitation, this is the mapping structure:
-   * LambdaInsightsVersions10980 // for version 1.0.98.0
-   * - us-east-1
-   * -- {'arn': 'arn1'},
-   * - us-east-2
-   * -- {'arn': 'arn2'}
-   * LambdaInsightsVersions10890 // a separate mapping version 1.0.89.0
-   * - us-east-1
-   * -- {'arn': 'arn3'},
-   * - us-east-2
-   * -- {'arn': 'arn4'}
-   */
-
-  const mapName = DEFAULT_MAPPING_PREFIX + insightsVersion.split('.').join('');
```

Comment:
I'm not seeing where this was replicated/replaced. If I'm reading the above comment correctly, that means the new mappings won't actually deploy.

Reply:
They will deploy, the mapping name will just be different.

Reply:
Ah, ok. FWIW, I checked out this branch locally and ran the aws-lambda tests, and got several failures (looks related to the rename).